### ISSUE TITLE
Disambiguated graph returns an object instead of a list for nested cards with cardinality 'n'.

### ISSUE THREAD

**Issue by apeters** [archesproject/arches#7616]
This only is an issue when the value of the node is null/None.  

To reproduce, make a resource graph with a nested card with Cardinality 'n'.
1. Create an instance of that graph but leave the card in question blank.
2. Visualise the instance as json and notice that the "@value" of that card is an object
3. Now add data to that card
4. Now notice that the "@value" of that card is a list

In the below example 'Child Card' is a card with cardinality 'n'.  I would expect it to be an empty array instead of an object with `@value` = null.

```
{
    "displaydescription": "undefined",
    "displayname": "undefined",
    "graph_id": "44b5dbb4-e8f9-11eb-89f0-acde48001122",
    "legacyid": null,
    "map_popup": "undefined",
    "resource": {
        "Name Node": {
            "@node_id": "9487e268-e8f9-11eb-89f0-acde48001122",
            "@tile_id": "30184358-a330-431b-95e4-f8dfd8a965e1",
            "@value": "TEST"
        },
        "String Node": {
            "@node_id": "59d0f920-e8f9-11eb-89f0-acde48001122",
            "@tile_id": "5e41aafa-f622-49cd-a6a6-31d8094ca13d",
            "@value": "NON_DATA_COLLECTING_NODE",
            "Child Card": {
                "@node_id": "dfef281a-e8f9-11eb-89f0-acde48001122",
                "@tile_id": "5e41aafa-f622-49cd-a6a6-31d8094ca13d",
                "@value": null
            },
            "NUMBER Node": {
                "@node_id": "6b2891be-e8fa-11eb-89f0-acde48001122",
                "@tile_id": "5e41aafa-f622-49cd-a6a6-31d8094ca13d",
                "@value": "1"
            }
        }
    },
    "resourceinstanceid": "342ec86c-e725-452f-ab46-0ec5150a989d"
}
```

**Comment by chiatt** [comment]
@apeters I think this is resolved, right?

**Comment by chiatt** [comment]
Confirmed that this is still a bug in the beta endpoint

**Comment by apeters** [comment]
@chrabyrd could you take a look at this?  @aarongundel just ran into this issue today and it would be nice to get this resolved soon.

**Comment by chrabyrd** [comment]
@apeters would this be considered fixed?

**Comment by chiatt** [comment]
Oops! I should have closed this when I merged #7939.
